src/phase0/configs/mainnet.rs

```diff
@@ -1,8 +1,10 @@
 use crate::primitives::{Epoch, Gwei, VersionBytes};
+use ssz_rs::Vector;
 
 pub const MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: u64 = 16384;
 pub const MIN_GENESIS_TIME: u64 = 1606824000;
 pub const GENESIS_FORK_VERSION_BYTES: VersionBytes = [0u8; 4];
+pub const GENESIS_FORK_VERSION: Vector<u8, 4> = Vector::from_iter(GENESIS_FORK_VERSION_BYTES);
```

Comment:
i had issues w/ this bc `from_iter` is not `const`... did I miss something? i'd prefer this if it works!

Reply:
I think you haven't pulled all updates